docs/README.md

```diff
@@ -1150,6 +1154,155 @@ In this setup there is an external control plane cluster (`cluster1`) and a remo
     kubectl delete ns sample --context="${CTX_CLUSTER2}"
     ```
 
+## Dual-stack Support
+
+Kubernetes supports dual-stack networking as a stable feature starting from
+[v1.23](https://kubernetes.io/docs/concepts/services-networking/dual-stack/), allowing clusters to handle both
+IPv4 and IPv6 traffic. With many cloud providers also beginning to offer dual-stack Kubernetes clusters, it's easier
+than ever to run services that function across both address types. Istio introduced dual-stack as an experimental
+feature in version 1.17, and it's expected to be promoted to [Alpha](https://github.com/istio/istio/issues/47998) in
+version 1.24. With Istio in dual-stack mode, services can communicate over both IPv4 and IPv6 endpoints, which helps
+organizations transition to IPv6 while still maintaining compatibility with their existing IPv4 infrastructure.
+
+When Kubernetes is configured for dual-stack, it automatically assigns an IPv4 and an IPv6 address to each pod,
+enabling them to communicate over both IP families. For services, however, you can control how they behave using
+the `ipFamilyPolicy` setting.
+
+Service.Spec.ipFamilyPolicy can take the following values
+- SingleStack: Only one IP family is configured for the service, which can be either IPv4 or IPv6.
+- PreferDualStack: Both IPv4 and IPv6 cluster IPs are assigned to the Service when dual-stack is enabled.
+                   However, if dual-stack is not enabled or supported, it falls back to singleStack behavior.
+- RequireDualStack: The service will be created only if both IPv4 and IPv6 addresses can be assigned.
+
+This allows you to specify the type of service, providing flexibility in managing your network configuration.
+For more details, you can refer to the Kubernetes [documentation](https://kubernetes.io/docs/concepts/services-networking/dual-stack/#services).
+
+### Prerequisites
+
+- Kubernetes 1.23 or later configured with dual-stack support.
+- Sail Operator is installed.
+- Kind cluster with dual-stack networking.
```

Comment:
nit: is it a prerequisite if we outline how to get one below? I'm fine either way